src/js/program/program-listeners.ts

```diff
@@ -166,11 +166,16 @@ export function mediaControllerListener<E extends keyof AllProviderEventsListene
             /* falls through to to trigger model event off model */
         case MEDIA_SEEK:
         case MEDIA_SEEKED:
-        case NATIVE_FULLSCREEN:
         case SUBTITLES_TRACKS:
         case 'subtitlesTracksData':
             model.trigger(type, event);
             break;
+        case NATIVE_FULLSCREEN:
+            if (model.get('pip')) {
```

Comment:
NATIVE_FULLSCREEN fires when going into PiP mode on Desktop Safari, resulting unwanted fullscreen on the player container.